recipe/meta.yaml

```diff
@@ -1,23 +1,21 @@
-{% set version = "1.3.0" %}
+{% set version = "1.4.0" %}
 
 package:
   name: ibis-framework
   version: {{ version }}
 
 source:
   url: https://github.com/ibis-project/ibis/archive/{{ version }}.tar.gz
-  sha256: 36e0b2694a4e88a1406756eb32b21df7a625009f83bdfed012ab0159c7b49117
+  sha256: dd3e3d6d495d4ed4f7e5b8ec523949bc54f6fc89f31ca60ef3dc103de8d37635
 
 build:
-  number: 7
+  number: 0
```

Comment:
maybe you should use something like `skip: true  # [py>=37]`

Reply:
ref: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html

Reply:
actually it should be `skip: true # [py<37]`

Reply:
Thanks, that was it. I tried before, but not sure why it deleted the whole matrix, I guess I made a typo or something.